features/support/helpers/api_docs_service_helper.rb

```diff
@@ -12,11 +13,17 @@ def fill_in_api_docs_service_body(value)
     find('.pf-c-page__main-section').click # HACK: need to click outside to lose focus
   end
 
+  def transform_swagger_version(spec_version)
+    match =  /(invalid)?\s?(Swagger 1.2|Swagger 2|OAS 3.0|OAS 3.1)/.match(spec_version)
+    { version: numbered_swagger_version(match[2]), invalid: match[1].present? }
+  end
+
   def numbered_swagger_version(version)
```

Comment:
hmm, shouldn't this method accept a named parameter `invalid` to properly work?

Reply:
well, this method just converts e.g. `Swagger 1.2` to `1.2`. Maybe it's confusing because it looks like in the invocation both are passed? But there is a `)` after `match[2]` on like 18 :wink: